packages/smart-wallet/src/smartWallet.js

```diff
@@ -124,6 +135,8 @@ const mapToRecord = map => Object.fromEntries(map.entries());
  *   purseBalances: MapStore<RemotePurse, Amount>,
  *   updatePublishKit: PublishKit<UpdateRecord>,
  *   currentPublishKit: PublishKit<CurrentWalletRecord>,
+ *   possiblyExitableOffers: MapStore<import('./offers.js').OfferId, import('./offers.js').OfferStatus>,
+ *   activeOfferSeats: MapStore<import('./offers.js').OfferId, UserSeat<unknown>>,
```

Comment:
I suppose we could store the seats in `possiblyExitableOffers` and filter them out when we publish `current`. I'm not sure how much of a performance difference it is.

Reply:
Good question. Compute vs storage, I don't know how those costs compare on the chain. My hunch is that we shouldn't be using durable storage as a cache of computed results but let's get more input.